unconvert.go

```diff
@@ -166,8 +167,9 @@ var (
 	flagApply      = flag.Bool("apply", false, "apply edits to source files")
 	flagCPUProfile = flag.String("cpuprofile", "", "write CPU profile to file")
 	// TODO(mdempsky): Better description and maybe flag name.
-	flagSafe = flag.Bool("safe", false, "be more conservative (experimental)")
-	flagV    = flag.Bool("v", false, "verbose output")
+	flagSafe      = flag.Bool("safe", false, "be more conservative (experimental)")
+	flagV         = flag.Bool("v", false, "verbose output")
+	flagBuildTags = flag.String("tags", "", "a space-separated list of build tags. see also 'go build' option for details")
```

Comment:
You might want to consider using [`buildutil.TagsFlag`](https://godoc.org/golang.org/x/tools/go/buildutil#TagsFlag) here.